dat-schema/_Core.gql

```diff
@@ -477,7 +483,7 @@ type BaseItemTypes {
   ItemThemesKey: ItemThemes
   IdentifyMagic_AchievementItemsKeys: [AchievementItems]
   FragmentBaseItemTypesKey: BaseItemTypes
-  _: bool
+  IsBlessing: bool
```

Comment:
Orb of unmaking and elevated sextant clearly not blessings

Reply:
Oh, yeah. I assumed the column name got lost somewhere.